tests/test_stream.py

```diff
@@ -1,10 +1,30 @@
+import json
+
 import pytest
+import responses
 
 from httpie.compat import is_windows
+from httpie.cli.constants import PRETTY_MAP
 from httpie.output.streams import BINARY_SUPPRESSED_NOTICE
+from httpie.plugins import ConverterPlugin
+from httpie.plugins.registry import plugin_manager
+
 from .utils import StdinBytesIO, http, MockEnvironment
 from .fixtures import BIN_FILE_CONTENT, BIN_FILE_PATH
 
+PRETTY_OPTIONS = list(PRETTY_MAP.keys())
+
+
+class FakeConverterPlugin(ConverterPlugin):
```

Comment:
It’s not _completely_ fake. It does something so I’d vote for giving it a slightly more meaningful name.